src/main/java/Game.java

```diff
@@ -50,6 +56,17 @@ public boolean setNumPlayers(int n){
     }
 
     public void generateHTMLFiles() {
+        String pathURL = Game.class.getResource("").getPath();
```

Comment:
Use a relative path, instead of getting the path to the resource files and modifying it.